pkg/api/types.go

```diff
@@ -505,14 +505,20 @@ func (l *VolumeList) Set(value string) error {
 		return errors.New("invalid format, must be source:destination")
 	}
 	var mount []string
-	pos := strings.LastIndex(value, ":")
+	var pos int
+	if strings.HasPrefix(value, "C:\\") {
```

Comment:
nm, i see you're not stripping the prefix.  but still the check needs to be looser.